library/src/main/resources/SAMLSpecRefMessage.properties

```diff
@@ -302,6 +302,22 @@ SAMLCore_8_3_6_b=[T]he content of the element is the identifier of an entity tha
   SAML profiles (such as a service provider supporting the browser SSO profile)...The syntax of \
   such an identifier is a URI of not more than 1024 characters in length.
 
+SAMLCore_8_3_7_a=Persistent name identifier values MUST NOT exceed a length of 256 characters.
+
+SAMLCore_8_3_7_b=The element's NameQualifier attribute, if present, MUST contain the unique \
+  identifier of the identity provider that generated the identifier (see Section 8.3.6).
+
+SAMLCore_8_3_7_c=Note that a different system entity might later issue its own protocol message or \
```

Comment:
❗️ We should add this to `NotTested`.

In the reasoning we should note that we can't tell if a message was generated from another SAML entity, or if the attribute was omitted all together.

Reply:
That was being added in a separate PR. I guess I can do it in this one